src/schema/amd-schema.ts

```diff
@@ -0,0 +1,175 @@
+            r1CheckComments: 'Value of r1CheckComments should be a string',
+            rcBuildComments: 'Value of rcBuildComments should be a string',
+            rcCheckComments: 'Value of rcCheckComments should be a string',
+        },
+        required: {
+            screenNumber: 'Missing required parameter: screenNumber',
+            activityTypeName: 'Missing required parameter: activityTypeName',
+            layoutInfo: 'Missing required parameter: layoutInfo',
+            rubric: 'Missing required parameter: rubric',
+            contents: 'Missing required parameter: contents',
+            supplementArea: 'Missing required parameter: supplementArea',
+            attachmentArea1: 'Missing required parameter: attachmentArea1',
+            attachmentArea2: 'Missing required parameter: attachmentArea2',
+            mediaInHeader: 'Missing required parameter: mediaInHeader',
+            mediaInHeaderScreenInfo: 'Missing required parameter: mediaInHeaderScreenInfo',
+            feedBackBand1Correct: 'Missing required parameter: feedBackBand1Correct',
+            feedBackBand2InCorrect: 'Missing required parameter: feedBackBand2InCorrect',
+            feedBackThreshold: 'Missing required parameter: feedBackThreshold',
+            loGlobalOptions: 'Missing required parameter: loGlobalOptions',
+            loScreenSpecificOptions: 'Missing required parameter: loScreenSpecificOptions',
+            additionalNotes: 'Missing required parameter: additionalNotes',
+            r1BuildComments: 'Missing required parameter: r1BuildComments',
+            r1CheckiComments: 'Missing required parameter: r1CheckiComments',
+            r2BuildComments: 'Missing required parameter: r2BuildComments',
+            r1CheckComments: 'Missing required parameter: r1CheckComments',
+            rcBuildComments: 'Missing required parameter: rcBuildComments',
+            rcCheckComments: 'Missing required parameter: rcCheckComments',
+        }
+    }
+};
+
+export const LoSchema = {
+    type: 'object',
+    properties: {
+        lo: { type: 'string' },
+        loTitle: { type: 'string' },
+        screen: {
+            "type": "array",
+            "items": ScreenSchema
+        },
+        loNumber: { type: 'number' },
+    },
+    required: [
+        'lo',
+        'loTitle',
+        'screen',
+        'loNumber'
+    ],
+    errorMessage: {
+        properties: {
+            lo: 'Value of lo should be a string',
+            loTitle: 'Value of loTitle should be a string',
+            screen: 'Value of screen should be an array of screens',
+            loNumber: 'Value of loNumber should be a number',
+        },
+        required: {
+            lo: 'Missing required parameter: lo',
+            loTitle: 'Missing required parameter: loTitle',
+            screen: 'Missing required parameter: screen',
+            loNumber: 'Missing required parameter: loNumber',
+        }
+    }
+};
+
+export const ProductInfoSchema = {
```

Comment:
Please check if there is need to export this schema as it is only being used in the default one